conf/application.conf

```diff
@@ -178,4 +178,6 @@ nunjucks {
   libPaths = ["govuk-frontend", "hmrc-frontend"]
 }
 
-
+pagination {
+  numberOfMovements = 20
```

Comment:
What about having 2 different config keys, one for arrivals and one for departures? They will most likely never differ but this gives us the flexibility to experiment with different values in future